SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+## Supported Versions
+
+Currently, we are providing security updates for the latest release in the v1.x series:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| Latest v1.x  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability in `libuv`, please use the [GitHub's private vulnerability reporting feature](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability) to report it to us.
+
+Please report the vulnerability directly in the `libuv` repository at https://github.com/libuv/libuv.
```

Comment:
This contradicts the previous paragraph (also line 26.)

Reply:
I understand it can be misleading, but it refers to the repo where the private vuln report so be created against. I'll make an update to make that clearer.

Reply:
Updated PTAL.